src/test/java/tests/FilterEvalTests.java

```diff
@@ -425,19 +425,39 @@ public void updated() {
         issue.setUpdatedAt(now.minusDays(2));
 
         assertEquals(false, matches("updated:<24", issue));
+        assertEquals(false, matches("updatedByOthers:<24", issue));
+        assertEquals(false, matches("updatedBySelf:<24", issue));
```

Comment:
These tests don't cover what was changed. You have to stub out each the test issue's metadata with events and comments, and ensure that the right qualifier matches in the right case. There are 2^3 cases to cover.